pkg/util/contracts/contracts.go

```diff
@@ -56,6 +58,19 @@ func (l SupportedContractImplementationsList) IsSupported(contract string, impl
 	return false
 }
 
+func (l SupportedContractImplementationsList) GetConfig(contract string, impl ContractImplementation) map[string]string {
```

Comment:
nts: TODO: Refactor everything in this file. Use maps instead of lists. Consolidate this function with the previous.